test/e2e/flask/multichain-api/testHelpers.ts

```diff
@@ -55,13 +72,29 @@ export async function openMultichainDappAndConnectWalletWithExternallyConnectabl
  *
  * @param driver - E2E test driver {@link Driver}, wrapping the Selenium WebDriver.
  * @param scopes - scopes to create session for.
+ * @param accounts - The account addresses to create session for.
  */
 export async function initCreateSessionScopes(
   driver: Driver,
   scopes: string[],
+  accounts: string[] = [],
 ): Promise<void> {
-  for (const scope of scopes) {
-    await driver.clickElement(`input[name="${scope}"]`);
+  for (const [i, scope] of scopes.entries()) {
+    const scopeInput = await driver.waitForSelector(`#custom-Scope-input-${i}`);
+
+    // @ts-expect-error Driver.findNestedElement injects `fill` method onto returned element, but typescript compiler will not let us access this method without a complaint, so we override it.
+    scopeInput.fill(scope);
+    await driver.clickElement(`#add-custom-scope-button-${i}`);
```

Comment:
not seeing these ids in the test-dapp itself?

Reply:
it's in the most recent test-dapp-multichain release